components/Project/Project.module.css

```diff
@@ -1,190 +1,190 @@
 .project {
-    padding: 40px 44px 44px 36px;
-    vertical-align: top;
-    box-shadow: 0 3px 12px -2px rgba(17, 12, 46, .15);
-    width: 100%;
-    break-inside: avoid;
-    background: white;
-    border-radius: 8px;
-    transition: .1s ease box-shadow;
+  padding: 40px 30px;
```

Comment:
nit
`40px 32px`

we haven't approved this practice yet
but we try to use multiples of 8 (or 4?)

Reply:
done